crates/teloxide-core/Cargo.toml

```diff
@@ -72,7 +72,7 @@ once_cell = "1.5.0"
 takecell = "0.1"
 take_mut = "0.2"
 rc-box = "1.1.1"
-chrono = { version = "0.4.30", default-features = false }
+chrono = { version = "~0.4.32", default-features = false }
```

Comment:
what is `~`?... I'm pretty sure you don't need it?...

Reply:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#tilde-requirements

It's equivalent in this case, so removed the `~`.